integration/kubernetes/alluxio-master.yaml

```diff
@@ -0,0 +1,81 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: alluxio-master
+  labels:
+    app: alluxio
+spec:
+  ports:
+  - port: 19998
+    name: rpc
+  - port: 19999
+    name: web
+  clusterIP: None
+  selector:
+    app: alluxio-master
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: alluxio-master
+spec:
+  selector:
+    matchLabels:
+      app: alluxio-master
+  serviceName: "alluxio-master"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: alluxio-master
+    spec:
+      hostNetwork: true
+      hostPID: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: alluxio-master
+          image: alluxio/alluxio:1.6.1
+          resources:
+            requests:
+              cpu: "0.5"
+              memory: "512M"
+            limits:
+              cpu: "1"
+              memory: "1024M"
+          command: ["/entrypoint.sh"]
+          args: ["master"]
+          envFrom:
+          - configMapRef:
+              name: alluxio-common-config
+          - configMapRef:
+              name: alluxio-master-config
+          ports:
+          - containerPort: 19998
+            name: rpc
+          - containerPort: 19999
+            name: web
+          volumeMounts:
+            - name: alluxio-journal
+              mountPath: /journal
+            - name: alluxio-domain
+              mountPath: /opt/domain
+      restartPolicy: Always
+      volumes:
+        - name: alluxio-journal
+          emptyDir:
```

Comment:
it's good for journal to be disk so that it persists across restart

Reply:
let me address this w/ HA in a follow up PR

Reply:
handling HA in a separate PR sgtm